lib/logthrdestdrv.c

```diff
@@ -121,6 +128,7 @@ _connect(LogThrDestDriver *self)
     }
 }
 
+/* NOTE: runs in the worker thread */
```

Comment:
These notes could be turned into asserts. A new assert could be created similar to main_loop_assert_main_thread.